ppl/zqd/core.go

```diff
@@ -93,38 +94,22 @@ func NewCore(ctx context.Context, conf Config) (*Core, error) {
 		}
 	}
 
-	router := mux.NewRouter()
-	router.Use(requestIDMiddleware())
-	router.Use(accessLogMiddleware(conf.Logger, "/status", "/metrics"))
-	router.Use(panicCatchMiddleware(conf.Logger))
-	router.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-		io.WriteString(w, indexPage)
-	})
-
-	debug := router.PathPrefix("/debug/pprof").Subrouter()
-	debug.HandleFunc("/cmdline", pprof.Cmdline)
-	debug.HandleFunc("/profile", pprof.Profile)
-	debug.HandleFunc("/symbol", pprof.Symbol)
-	debug.HandleFunc("/trace", pprof.Trace)
-	debug.PathPrefix("/").HandlerFunc(pprof.Index)
-
-	router.Handle("/metrics", promhttp.HandlerFor(registry, promhttp.HandlerOpts{}))
-	router.HandleFunc("/status", func(w http.ResponseWriter, r *http.Request) {
-		io.WriteString(w, "ok")
-	})
-	router.HandleFunc("/version", func(w http.ResponseWriter, r *http.Request) {
-		w.Header().Add("Content-Type", "application/json")
-		json.NewEncoder(w).Encode(&api.VersionResponse{Version: conf.Version})
-	})
```

Comment:
I'd just as soon keep this stuff here as move it to `addAuxilaryRoutes`, but either's fine.